rrbot_moveit_demo_nodes/src/run_moveit_cpp.cpp

```diff
@@ -0,0 +1,117 @@
+/*********************************************************************
```

Comment:
Does it make sense to copy this code into here? Or depend on it from the moveit project? It will likely get out of sync here

Reply:
the group name was hardcoded in the moveit2 tutorials, I decided to move this here.

Reply:
maybe we should update the moveit tutorial to not hardcode the group name... @henningkayser @JafarAbdi what do you think?